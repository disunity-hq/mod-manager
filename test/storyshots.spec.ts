import initStoryshots from '@storybook/addon-storyshots';

import { remote } from 'electron';

// remote.getCurrentWindow().show();
console.log('foobar');

initStoryshots({

});
