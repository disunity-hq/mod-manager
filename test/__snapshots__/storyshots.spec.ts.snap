// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Test Test Header 1`] = `
<h1>
  This is a test.
</h1>
`;

exports[`Storyshots Window NavBar 1`] = `
<section
  className="ant-layout ant-layout-has-sider"
  style={
    Object {
      "minHeight": "calc(100vh - 48px)",
    }
  }
>
  <aside
    className="ant-layout-sider ant-layout-sider-dark ant-layout-sider-has-trigger"
    style={
      Object {
        "flex": "0 0 200px",
        "maxWidth": "200px",
        "minWidth": "200px",
        "width": "200px",
      }
    }
  >
    <div
      className="ant-layout-sider-children"
    >
      <div
        className="logo"
      />
      <ul
        className="ant-menu ant-menu-dark ant-menu-root ant-menu-inline"
        onMouseEnter={[Function]}
        onTransitionEnd={[Function]}
        role="menu"
        style={Object {}}
      >
        <li
          className="ant-menu-submenu ant-menu-submenu-inline ant-menu-submenu-open"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
        >
          <div
            aria-expanded={true}
            aria-haspopup="true"
            aria-owns="gamesSubMenu$Menu"
            className="ant-menu-submenu-title"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "paddingLeft": 24,
              }
            }
          >
            <span>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-gamepad fa-w-20 fa-1x "
                color="white"
                data-icon="gamepad"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={
                  Object {
                    "marginRight": "10px",
                  }
                }
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M480 96H160C71.6 96 0 167.6 0 256s71.6 160 160 160c44.8 0 85.2-18.4 114.2-48h91.5c29 29.6 69.5 48 114.2 48 88.4 0 160-71.6 160-160S568.4 96 480 96zM256 276c0 6.6-5.4 12-12 12h-52v52c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-52H76c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h52v-52c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h52c6.6 0 12 5.4 12 12v40zm184 68c-26.5 0-48-21.5-48-48s21.5-48 48-48 48 21.5 48 48-21.5 48-48 48zm80-80c-26.5 0-48-21.5-48-48s21.5-48 48-48 48 21.5 48 48-21.5 48-48 48z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
              <span>
                My Games
              </span>
            </span>
            <i
              className="ant-menu-submenu-arrow"
            />
          </div>
          <ul
            className="ant-menu  ant-menu-sub ant-menu-inline"
            id="gamesSubMenu$Menu"
            role="menu"
            style={Object {}}
          >
            <li
              className="ant-menu-item"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              role="menuitem"
              style={
                Object {
                  "paddingLeft": 48,
                }
              }
            >
              Risk of Rain 2
            </li>
            <li
              className="ant-menu-item"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              role="menuitem"
              style={
                Object {
                  "paddingLeft": 48,
                }
              }
            >
              <i
                aria-label="icon: plus"
                className="anticon anticon-plus"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="plus"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z"
                  />
                  <path
                    d="M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z"
                  />
                </svg>
              </i>
              <span>
                Add Game
              </span>
            </li>
          </ul>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <i
            aria-label="icon: search"
            className="anticon anticon-search"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="search"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0 0 11.6 0l43.6-43.5a8.2 8.2 0 0 0 0-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
              />
            </svg>
          </i>
          <span>
            Browse Mods
          </span>
        </li>
        <li
          className="ant-menu-submenu ant-menu-submenu-inline"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
        >
          <div
            aria-expanded={false}
            aria-haspopup="true"
            className="ant-menu-submenu-title"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "paddingLeft": 24,
              }
            }
          >
            <span>
              <i
                aria-label="icon: setting"
                className="anticon anticon-setting"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="setting"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M924.8 625.7l-65.5-56c3.1-19 4.7-38.4 4.7-57.8s-1.6-38.8-4.7-57.8l65.5-56a32.03 32.03 0 0 0 9.3-35.2l-.9-2.6a443.74 443.74 0 0 0-79.7-137.9l-1.8-2.1a32.12 32.12 0 0 0-35.1-9.5l-81.3 28.9c-30-24.6-63.5-44-99.7-57.6l-15.7-85a32.05 32.05 0 0 0-25.8-25.7l-2.7-.5c-52.1-9.4-106.9-9.4-159 0l-2.7.5a32.05 32.05 0 0 0-25.8 25.7l-15.8 85.4a351.86 351.86 0 0 0-99 57.4l-81.9-29.1a32 32 0 0 0-35.1 9.5l-1.8 2.1a446.02 446.02 0 0 0-79.7 137.9l-.9 2.6c-4.5 12.5-.8 26.5 9.3 35.2l66.3 56.6c-3.1 18.8-4.6 38-4.6 57.1 0 19.2 1.5 38.4 4.6 57.1L99 625.5a32.03 32.03 0 0 0-9.3 35.2l.9 2.6c18.1 50.4 44.9 96.9 79.7 137.9l1.8 2.1a32.12 32.12 0 0 0 35.1 9.5l81.9-29.1c29.8 24.5 63.1 43.9 99 57.4l15.8 85.4a32.05 32.05 0 0 0 25.8 25.7l2.7.5a449.4 449.4 0 0 0 159 0l2.7-.5a32.05 32.05 0 0 0 25.8-25.7l15.7-85a350 350 0 0 0 99.7-57.6l81.3 28.9a32 32 0 0 0 35.1-9.5l1.8-2.1c34.8-41.1 61.6-87.5 79.7-137.9l.9-2.6c4.5-12.3.8-26.3-9.3-35zM788.3 465.9c2.5 15.1 3.8 30.6 3.8 46.1s-1.3 31-3.8 46.1l-6.6 40.1 74.7 63.9a370.03 370.03 0 0 1-42.6 73.6L721 702.8l-31.4 25.8c-23.9 19.6-50.5 35-79.3 45.8l-38.1 14.3-17.9 97a377.5 377.5 0 0 1-85 0l-17.9-97.2-37.8-14.5c-28.5-10.8-55-26.2-78.7-45.7l-31.4-25.9-93.4 33.2c-17-22.9-31.2-47.6-42.6-73.6l75.5-64.5-6.5-40c-2.4-14.9-3.7-30.3-3.7-45.5 0-15.3 1.2-30.6 3.7-45.5l6.5-40-75.5-64.5c11.3-26.1 25.6-50.7 42.6-73.6l93.4 33.2 31.4-25.9c23.7-19.5 50.2-34.9 78.7-45.7l37.9-14.3 17.9-97.2c28.1-3.2 56.8-3.2 85 0l17.9 97 38.1 14.3c28.7 10.8 55.4 26.2 79.3 45.8l31.4 25.8 92.8-32.9c17 22.9 31.2 47.6 42.6 73.6L781.8 426l6.5 39.9zM512 326c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm79.2 255.2A111.6 111.6 0 0 1 512 614c-29.9 0-58-11.7-79.2-32.8A111.6 111.6 0 0 1 400 502c0-29.9 11.7-58 32.8-79.2C454 401.6 482.1 390 512 390c29.9 0 58 11.6 79.2 32.8A111.6 111.6 0 0 1 624 502c0 29.9-11.7 58-32.8 79.2z"
                  />
                </svg>
              </i>
              <span>
                Settings
              </span>
            </span>
            <i
              className="ant-menu-submenu-arrow"
            />
          </div>
          <div />
        </li>
        <li
          className="ant-menu-submenu ant-menu-submenu-inline"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
        >
          <div
            aria-expanded={false}
            aria-haspopup="true"
            className="ant-menu-submenu-title"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "paddingLeft": 24,
              }
            }
          >
            <span>
              <i
                aria-label="icon: team"
                className="anticon anticon-team"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="team"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M824.2 699.9a301.55 301.55 0 0 0-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 0 0-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 0 0 8 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 0 1 612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 0 0 8-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 0 1-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 0 1 612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 0 1-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 0 0 8 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                  />
                </svg>
              </i>
              <span>
                Team
              </span>
            </span>
            <i
              className="ant-menu-submenu-arrow"
            />
          </div>
          <div />
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <i
            aria-label="icon: file"
            className="anticon anticon-file"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="file"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M854.6 288.6L639.4 73.4c-6-6-14.1-9.4-22.6-9.4H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V311.3c0-8.5-3.4-16.7-9.4-22.7zM790.2 326H602V137.8L790.2 326zm1.8 562H232V136h302v216a42 42 0 0 0 42 42h216v494z"
              />
            </svg>
          </i>
          <span>
            File
          </span>
        </li>
      </ul>
    </div>
    <div
      className="ant-layout-sider-trigger"
      onClick={[Function]}
      style={
        Object {
          "width": "200px",
        }
      }
    >
      <i
        aria-label="icon: left"
        className="anticon anticon-left"
      >
        <svg
          aria-hidden="true"
          className=""
          data-icon="left"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 0 0 0 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
          />
        </svg>
      </i>
    </div>
  </aside>
  <section
    className="ant-layout ant-layout-has-sider"
  >
    <aside
      className="ant-layout-sider ant-layout-sider-dark ant-layout-sider-has-trigger"
      style={
        Object {
          "flex": "0 0 200px",
          "maxWidth": "200px",
          "minWidth": "200px",
          "width": "200px",
        }
      }
    >
      <div
        className="ant-layout-sider-children"
      >
        <ul
          className="ant-menu ant-menu-dark ant-menu-root ant-menu-inline"
          onMouseEnter={[Function]}
          onTransitionEnd={[Function]}
          role="menu"
          style={Object {}}
        >
          <li
            className="ant-menu-submenu ant-menu-submenu-inline ant-menu-submenu-open"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            role="menuitem"
          >
            <div
              aria-expanded={true}
              aria-haspopup="true"
              aria-owns="gamesSubMenu$Menu"
              className="ant-menu-submenu-title"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "paddingLeft": 24,
                }
              }
            >
              <span>
                <i
                  aria-label="icon: folder"
                  className="anticon anticon-folder"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="folder"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M880 298.4H521L403.7 186.2a8.15 8.15 0 0 0-5.5-2.2H144c-17.7 0-32 14.3-32 32v592c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V330.4c0-17.7-14.3-32-32-32zM840 768H184V256h188.5l119.6 114.4H840V768z"
                    />
                  </svg>
                </i>
                <span>
                  My Games
                </span>
              </span>
              <i
                className="ant-menu-submenu-arrow"
              />
            </div>
            <ul
              className="ant-menu  ant-menu-sub ant-menu-inline"
              id="gamesSubMenu$Menu"
              role="menu"
              style={Object {}}
            >
              <li
                className="ant-menu-item"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                role="menuitem"
                style={
                  Object {
                    "paddingLeft": 48,
                  }
                }
              >
                Risk of Rain 2
              </li>
              <li
                className="ant-menu-item"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                role="menuitem"
                style={
                  Object {
                    "paddingLeft": 48,
                  }
                }
              >
                <i
                  aria-label="icon: plus"
                  className="anticon anticon-plus"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="plus"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z"
                    />
                    <path
                      d="M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z"
                    />
                  </svg>
                </i>
                <span>
                  Add Game
                </span>
              </li>
            </ul>
          </li>
          <li
            className="ant-menu-item"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            role="menuitem"
            style={
              Object {
                "paddingLeft": 24,
              }
            }
          >
            <i
              aria-label="icon: search"
              className="anticon anticon-search"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="search"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0 0 11.6 0l43.6-43.5a8.2 8.2 0 0 0 0-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                />
              </svg>
            </i>
            <span>
              Browse Mods
            </span>
          </li>
          <li
            className="ant-menu-submenu ant-menu-submenu-inline"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            role="menuitem"
          >
            <div
              aria-expanded={false}
              aria-haspopup="true"
              className="ant-menu-submenu-title"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "paddingLeft": 24,
                }
              }
            >
              <span>
                <i
                  aria-label="icon: setting"
                  className="anticon anticon-setting"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="setting"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M924.8 625.7l-65.5-56c3.1-19 4.7-38.4 4.7-57.8s-1.6-38.8-4.7-57.8l65.5-56a32.03 32.03 0 0 0 9.3-35.2l-.9-2.6a443.74 443.74 0 0 0-79.7-137.9l-1.8-2.1a32.12 32.12 0 0 0-35.1-9.5l-81.3 28.9c-30-24.6-63.5-44-99.7-57.6l-15.7-85a32.05 32.05 0 0 0-25.8-25.7l-2.7-.5c-52.1-9.4-106.9-9.4-159 0l-2.7.5a32.05 32.05 0 0 0-25.8 25.7l-15.8 85.4a351.86 351.86 0 0 0-99 57.4l-81.9-29.1a32 32 0 0 0-35.1 9.5l-1.8 2.1a446.02 446.02 0 0 0-79.7 137.9l-.9 2.6c-4.5 12.5-.8 26.5 9.3 35.2l66.3 56.6c-3.1 18.8-4.6 38-4.6 57.1 0 19.2 1.5 38.4 4.6 57.1L99 625.5a32.03 32.03 0 0 0-9.3 35.2l.9 2.6c18.1 50.4 44.9 96.9 79.7 137.9l1.8 2.1a32.12 32.12 0 0 0 35.1 9.5l81.9-29.1c29.8 24.5 63.1 43.9 99 57.4l15.8 85.4a32.05 32.05 0 0 0 25.8 25.7l2.7.5a449.4 449.4 0 0 0 159 0l2.7-.5a32.05 32.05 0 0 0 25.8-25.7l15.7-85a350 350 0 0 0 99.7-57.6l81.3 28.9a32 32 0 0 0 35.1-9.5l1.8-2.1c34.8-41.1 61.6-87.5 79.7-137.9l.9-2.6c4.5-12.3.8-26.3-9.3-35zM788.3 465.9c2.5 15.1 3.8 30.6 3.8 46.1s-1.3 31-3.8 46.1l-6.6 40.1 74.7 63.9a370.03 370.03 0 0 1-42.6 73.6L721 702.8l-31.4 25.8c-23.9 19.6-50.5 35-79.3 45.8l-38.1 14.3-17.9 97a377.5 377.5 0 0 1-85 0l-17.9-97.2-37.8-14.5c-28.5-10.8-55-26.2-78.7-45.7l-31.4-25.9-93.4 33.2c-17-22.9-31.2-47.6-42.6-73.6l75.5-64.5-6.5-40c-2.4-14.9-3.7-30.3-3.7-45.5 0-15.3 1.2-30.6 3.7-45.5l6.5-40-75.5-64.5c11.3-26.1 25.6-50.7 42.6-73.6l93.4 33.2 31.4-25.9c23.7-19.5 50.2-34.9 78.7-45.7l37.9-14.3 17.9-97.2c28.1-3.2 56.8-3.2 85 0l17.9 97 38.1 14.3c28.7 10.8 55.4 26.2 79.3 45.8l31.4 25.8 92.8-32.9c17 22.9 31.2 47.6 42.6 73.6L781.8 426l6.5 39.9zM512 326c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm79.2 255.2A111.6 111.6 0 0 1 512 614c-29.9 0-58-11.7-79.2-32.8A111.6 111.6 0 0 1 400 502c0-29.9 11.7-58 32.8-79.2C454 401.6 482.1 390 512 390c29.9 0 58 11.6 79.2 32.8A111.6 111.6 0 0 1 624 502c0 29.9-11.7 58-32.8 79.2z"
                    />
                  </svg>
                </i>
                <span>
                  Settings
                </span>
              </span>
              <i
                className="ant-menu-submenu-arrow"
              />
            </div>
            <div />
          </li>
          <li
            className="ant-menu-submenu ant-menu-submenu-inline"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            role="menuitem"
          >
            <div
              aria-expanded={false}
              aria-haspopup="true"
              className="ant-menu-submenu-title"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "paddingLeft": 24,
                }
              }
            >
              <span>
                <i
                  aria-label="icon: team"
                  className="anticon anticon-team"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="team"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M824.2 699.9a301.55 301.55 0 0 0-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 0 0-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 0 0 8 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 0 1 612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 0 0 8-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 0 1-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 0 1 612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 0 1-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 0 0 8 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                    />
                  </svg>
                </i>
                <span>
                  Team
                </span>
              </span>
              <i
                className="ant-menu-submenu-arrow"
              />
            </div>
            <div />
          </li>
          <li
            className="ant-menu-item"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            role="menuitem"
            style={
              Object {
                "paddingLeft": 24,
              }
            }
          >
            <i
              aria-label="icon: file"
              className="anticon anticon-file"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="file"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M854.6 288.6L639.4 73.4c-6-6-14.1-9.4-22.6-9.4H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V311.3c0-8.5-3.4-16.7-9.4-22.7zM790.2 326H602V137.8L790.2 326zm1.8 562H232V136h302v216a42 42 0 0 0 42 42h216v494z"
                />
              </svg>
            </i>
            <span>
              File
            </span>
          </li>
        </ul>
      </div>
      <div
        className="ant-layout-sider-trigger"
        onClick={[Function]}
        style={
          Object {
            "width": "200px",
          }
        }
      >
        <i
          aria-label="icon: left"
          className="anticon anticon-left"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="left"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 0 0 0 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
            />
          </svg>
        </i>
      </div>
    </aside>
    <main
      className="ant-layout-content"
      style={
        Object {
          "margin": "0 16px",
        }
      }
    >
      <section
        className="ant-layout"
      >
        <main
          className="ant-layout-content"
        >
          <div
            className="ant-breadcrumb"
            style={
              Object {
                "margin": "16px 0",
              }
            }
          >
            <span>
              <span
                className="ant-breadcrumb-link"
              >
                User
              </span>
              <span
                className="ant-breadcrumb-separator"
              >
                /
              </span>
            </span>
            <span>
              <span
                className="ant-breadcrumb-link"
              >
                Bill
              </span>
              <span
                className="ant-breadcrumb-separator"
              >
                /
              </span>
            </span>
          </div>
          <section
            className="ant-layout"
          >
            <h1>
              Index
            </h1>
            <button
              className="ant-btn"
              onClick={[Function]}
              type="button"
            >
              <span>
                Test Page
              </span>
            </button>
          </section>
        </main>
        <footer
          className="ant-layout-footer"
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          Disunity
        </footer>
      </section>
    </main>
  </section>
</section>
`;
